src/main/java/org/openmrs/isanteplus/performancedata/generator/service/GeneratorService.java

```diff
@@ -0,0 +1,33 @@
+package org.openmrs.isanteplus.performancedata.generator.service;
+
+import org.openmrs.isanteplus.performancedata.generator.service.impl.PatientGeneratorService;
+import org.openmrs.isanteplus.performancedata.model.AbstractEntity;
+import org.openmrs.isanteplus.performancedata.model.connection.Inserter;
+import org.openmrs.isanteplus.performancedata.options.GeneratorOptions;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
+import java.sql.SQLException;
+import java.util.Set;
+
+@Service
+public class GeneratorService {
+
+    @Inject
+    private PatientGeneratorService patientService;
```

Comment:
Rename the var, since I immediately associate this with the OpenMRS service

Reply:
patientGeneratorService instead of patientService will be ok?